internal/core/command/types.go

```diff
@@ -49,3 +52,13 @@ func (sc serviceCommand) Execute() (string, int, error) {
 	}
 	return buf.String(), resp.StatusCode, nil
 }
+
+func NewServiceCommand(device contract.Device, caller internal.HttpCaller, req *http.Request,
```

Comment:
Package public function returning a package private struct.  I think this should be `newServiceCommand`.